<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlPanelClass</class>
 <widget class="QWidget" name="ControlPanelClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ControlPanel</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QStackedWidget" name="controlStackedWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="loaderPage">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="initBtn">
            <property name="text">
             <string>初始化</string>
            </property>
            <property name="icon">
             <iconset resource="ControlPanel.qrc">
              <normaloff>:/resources/start.png</normaloff>:/resources/start.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>72</width>
              <height>72</height>
             </size>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextUnderIcon</enum>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="procedurePage">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QToolButton" name="safeProcBtn">
            <property name="text">
             <string>安全的过程</string>
            </property>
            <property name="icon">
             <iconset resource="ControlPanel.qrc">
              <normaloff>:/resources/safeProcedure.png</normaloff>:/resources/safeProcedure.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>72</width>
              <height>72</height>
             </size>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextUnderIcon</enum>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="unsafeProcBtn">
            <property name="text">
             <string>不安全的过程</string>
            </property>
            <property name="icon">
             <iconset resource="ControlPanel.qrc">
              <normaloff>:/resources/unsafeProcedure.png</normaloff>:/resources/unsafeProcedure.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>72</width>
              <height>72</height>
             </size>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextUnderIcon</enum>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="attackPage">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="attackBtn">
            <property name="text">
             <string>开始攻击</string>
            </property>
            <property name="icon">
             <iconset resource="ControlPanel.qrc">
              <normaloff>:/resources/malware.png</normaloff>:/resources/malware.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>72</width>
              <height>72</height>
             </size>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextUnderIcon</enum>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="helperWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="helperLabel">
           <property name="text">
            <string>帮助：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextBrowser" name="helperTextBrowser"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="outputWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="outputLabel">
        <property name="text">
         <string>状态输出：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="outputTextBrowser"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ControlPanel.qrc"/>
 </resources>
 <connections/>
</ui>
